MotorMove.py

```diff
@@ -0,0 +1,29 @@
+import RPi.GPIO as io
+
+class Motor:
+
+    def __init__(self, dirPinNum, motorPinNum):
+        self.dirPinNum = dirPinNum
+        self.motorPinNum = motorPinNum
+
+        io.setwarnings(False)
+        io.setmode(io.BOARD)
+        io.setup(self.motorPinNum, io.OUT)
+        io.setup(self.dirPinNum, io.OUT)
+        self.motor = io.PWM(self.motorPinNum, 100)
+        self.motor.stop()
+
+    def SetSpeed(self, speed):
+        if speed == 0: self.motor.stop(); return
+        dirSet = io.HIGH if speed < 0 else io.LOW
+        speedAbs = abs(speed)
+        self.motor.ChangeFrequency(speedAbs)
+        self.motor.start(50)
+
+
+if __name__ == "__main__":
+    M = Motor(10, 11)
```

Comment:
We need a way to limit speed. There is a min and max that is allowed to be set right? For instance we can send 1000 because that will exceed the speed that the motor can move. We should also have constants for all of the pins above.

Reply:
Commit 7fed4e32e269cf32748a9d706272f3b21858a4c6 changed the way we set speed by only allowing the MovementManager to set the frequency. A max and min should not be necessary now.